network/src/connectivity_manager/mod.rs

```diff
@@ -58,24 +59,31 @@ use std::{
     sync::Arc,
     time::Duration,
 };
-use tokio::{time, time::Instant};
 use tokio_retry::strategy::jitter;
 
 pub mod builder;
 #[cfg(test)]
 mod test;
 
+/// In addition to the backoff strategy, we also add some small random jitter to
+/// the delay before each dial. This jitter helps reduce the probability of
+/// simultaneous dials, especially in non-production environments where most nodes
+/// are spun up around the same time. Similarly, it smears the dials out in time
+/// to avoid spiky load / thundering herd issues where all dial requests happen
+/// around the same time at startup.
+const MAX_CONNECTION_DELAY_JITTER: Duration = Duration::from_millis(100);
```

Comment:
A much more apt description, probably could have just been succinct and said,"Helps prevent probability of simultaneous dials by spreading out dials". Also, a relevant post about jitter https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/